# Define the goal
geometry_msgs/PoseStamped target_grasp_pose
---
# Define the result
---
# Define the feedback
